manufacturing-ai-suite/wind-turbine-anomaly-detection/time_series_analytics_microservice/udfs/windturbine_anomaly_detector.py

```diff
@@ -4,36 +4,34 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-""" Custom user defined function for anomaly detection on the windturbine speed and generated power data. """
+""" Custom user defined function for anomaly detection on 
+the windturbine speed and generated power data. """
 
 import os
-import sys
-import json
-from collections import deque
-from kapacitor.udf.agent import Agent, Handler, Server
-from kapacitor.udf import udf_pb2
-import signal
-import stat
 import logging
-import tempfile
 import pickle
+import time
 import math
+import warnings
+from collections import deque
+from kapacitor.udf.agent import Agent, Handler
+from kapacitor.udf import udf_pb2
 import numpy as np
-from sklearn.ensemble import RandomForestRegressor
-from sklearn.linear_model import LinearRegression
-from sklearnex import patch_sklearn
-patch_sklearn()
-# import paho.mqtt.client as mqtt
-import modin.pandas as pd
-import datetime
-import time
 import requests
+from sklearnex import patch_sklearn, config_context
+patch_sklearn()
+from sklearn.linear_model import LinearRegression
+
+warnings.filterwarnings(
+    "ignore",
+    message=".*Threading.*parallel backend is not supported by Extension for Scikit-learn.*"
+)
```

Comment:
Hey @sathyendranv ,
Ideally this PR should only have this change, if branch stategy from forked repo to here is not working then maybe we can just cherry pick the commit into a branch (spawned from a main) and raise the PR.

In that way, it will be lot simpler.